client/hotrod-client-new/src/main/java/org/infinispan/client/hotrod/configuration/package-info.java

```diff
@@ -80,6 +80,12 @@
  *          <td>The {@link org.infinispan.client.hotrod.configuration.ConfigurationBuilder#connectionTimeout(int) timeout} for connections</td>
  *       </tr>
  *       <tr>
+ *          <td><b>infinispan.client.hotrod.server_failed_timeout</b></td>
```

Comment:
can we align the property name with the method name
```suggestion
 *          <td><b>infinispan.client.hotrod.server_failure_timeout</b></td>
```
Also, I'm unsure if `timeout` is the best word to describe since it is more like an expiration. I don't have any better suggestion so I'm fine to keep the `timeout` in the name.

Reply:
Yeah, I hate naming things.. we can always rename it later if we want before the Final release.